aldryn_reversion/tests/test_admin_views.py

```diff
@@ -359,6 +360,24 @@ def test_revision_view_reverts_object_to_selected_state(self):
             pk=self.simple_registered.pk)
         self.assertEquals(self.simple_registered.position, initial_position)
 
+    def test_admin_create_obj_view(self):
+        """Test that admin create view works and actually creates an object"""
+        obj_count = SimpleRegistered.objects.count()
+        obj = SimpleRegistered.objects.first()
+        url = reverse(
+            'admin:{0}_{1}_{2}'.format(
+                obj._meta.app_label,
+                obj._meta.model_name,
+                'add'))
+        client = Client()
+        # TODO: we can replace this with force_login() on Django 1.9+
+        client.login(username=self.super_user.username,
+                     password=self.super_user_password)
+        position = '777'
+        response = client.post(url, data={'position': position}, follow=True)
```

Comment:
I do not need a request object, but just to be able to perform a POST request to a view. Also, previously there was a missing is_staff flag on superuser that did not allow him to login into admin, but now he can.